modules/searchbar/searchbar-ios.js

```diff
@@ -34,12 +34,17 @@ export class SearchBar extends React.Component<Props> {
 		this._ref = ref
 	}
 
+	onHandleBlur = () => {
+		this._ref && this._ref.unFocus()
```

Comment:
Calling `unFocus` here instead of `unBlur`

> 
> * `blur()` - uses the iOS endEditing:true method on the underlying UISearchBar view.
> * `unFocus()` - calls resignFirstResponder on the UITextField used by the UISearchBar.